core/snippets/snippets/javascript.snippet

```diff
@@ -24,12 +24,6 @@ phrase: import star
 import * as $0 from "$0";
```

Comment:
While we are here, I'd be inclined to move importStarStatement into the new import.snippet file as well since python supports it

`from library_name import *`

Reply:
Done